functional-tests/README.md

```diff
@@ -19,6 +19,8 @@ brew install bitcoin
 curl -fsSLO --proto "=https" --tlsv1.2 https://bitcoin.org/bin/bitcoin-core-27.0/bitcoin-27.0-x86_64-linux-gnu.tar.gz
 tar xzf bitcoin-27.0-x86_64-linux-gnu.tar.gz
 sudo install -m 0755 -t /usr/local/bin bitcoin-27.0/bin/*
+# remove unarchived files, as we just copied it to /bin
+rm -rf bitcoin-27.0
```

Comment:
We should update this to v28.
Could you do it in this PR?
You can steal from https://github.com/alpenlabs/strata/blob/d8b1013af7224dea3785441b23a4004b607b3d16/.github/workflows/functional.yml#L64-L75